web_console/web_terminal/configuring-web-terminal.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="configuring-web-terminal"]
+= Configuring the web terminal
+include::_attributes/common-attributes.adoc[]
+:context: configuring-web-terminal
+
+toc::[]
+
+You can configure timeout and image settings for the web terminal, either for your current session, or for all user sessions if you are a cluster administrator.
```

Comment:
```suggestion
You can configure timeout and image settings for the web terminal, either for your current session or for all user sessions if you are a cluster administrator.
```
- No comma necessary because the `or` is not a coordinating conjunction here.